fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCSchemaResolver.java

```diff
@@ -90,7 +90,8 @@ public List<Column> convertToSRTable(ResultSet columnSet) throws SQLException {
                     columnSet.getInt("COLUMN_SIZE"),
                     columnSet.getInt("DECIMAL_DIGITS"));
             fullSchema.add(new Column(columnSet.getString("COLUMN_NAME"), type,
-                    columnSet.getString("IS_NULLABLE").equals("YES")));
+                    columnSet.getString("IS_NULLABLE").equals("YES"),
+                    columnSet.getString("REMARKS")));
```

Comment:
If columnSet.getString ("REMARKS") is null,  is this a safe behavior?

Reply:
I see that the old version code use an empty string "" as comment default value.
I will change the code to keep the same logic when columnSet.getString("REMARKS") is null.

Reply:
Done!

Reply:
`Throws:
SQLException – if the columnLabel is not valid; if a database access error occurs or this method is called on a closed result set`
will this interface throw SQLException exception under different db?

Reply:
I had verify It works for Mysql, PostgreSQL, Sqlserver.
I will add try-cache to prevent error when the metadata of some unknow database without REMARKS.